plugins/include/vault.inc

```diff
@@ -16,16 +16,45 @@
 #endif
 #define _vault_included
 
-/* Reads a data from given key.
-* If len is set to zero then get_vaultdata
-* returns value as an number. */
+ /**
+ * Reads data from a given key.
+ *
+ *
+ * @param key           Key to get the value from
+ * @param data          Buffer to copy the value to
+ * @param len           Buffer size. If len is set to 0 then the function will 
+ * 			        	 return the value as a number
+ *
+ * @return              If len is not 0 function will return the number of characters written.
+					 If len is 0 function will return the numerical value associated with the key.	
```

Comment:
Missing "the" before "function". Same on the line above it.

Reply:
Fixed